request_handlers_test.cc

```diff
@@ -1,13 +1,18 @@
 #include "gtest/gtest.h"
 #include "request_handlers.h"
 
+#include <cstddef>
+#include <fstream>
+
 class RequestHandlersTest : public ::testing::Test {
 protected:
+
   const size_t header_size = 45;
-  bool HandleRequest(const char* request_buffer,
-                     const size_t& request_buffer_size,
-                     char* &response_buffer,
-                     size_t& response_buffer_size) {
+
+  bool EchoHandleRequest(const char* request_buffer,
```

Comment:
We definitely want to use the HandleRequest + RequestRouter interface. This is addressed by PR #43.